function/helpers.go

```diff
@@ -37,3 +37,22 @@ func NewJSFetcthOptionArg() JSFetchOptionsArg {
 	}
 	return defaultOptions
 }
+
+type JSSendMailArg struct {
+	From     string `json:"from"`
+	To       string `json:"to"`
+	Subject  string `json:"subject"`
+	HTMLBody string `json:"htmlBody"`
+	TextBody string `json:"textBody"`
+}
+
+func NewJSSendMailArg() JSSendMailArg {
```

Comment:
if we're to keep the `JSSendMailArg` I'd remove this since it's equivalent to just doing: `data := JSSendMailArg{}`

Reply:
Agree